python/lsst/cp/pipe/cpBfk.py

```diff
@@ -440,34 +445,43 @@ def run(self, inputPtc, dummy, camera, inputDims):
                 # Get a mask for a 3px picture frame
                 valid = True
                 if kernelSize >= 7:
+                    # Last 1px
```

Comment:
I think this needs an explanatory comment.  We use a 3-pixel radius "annulus" to calculate the threshold, but then only check the 1-pixel outer frame?